sensitivity-api/src/main/java/com/powsybl/sensitivity/SensitivityComputationResults.java

```diff
@@ -41,23 +51,34 @@
     @JsonProperty("values")
     private final List<SensitivityValue> sensitivityValues;
 
+    @JsonProperty("contingencies present")
+    private final boolean contingenciesPresent;
+
+    @JsonProperty("contingencies values")
+    private final Map<String, List<SensitivityValue>> sensitivityValuesContingencies;
+
     /**
      * Hades2 sensitivity computation results
      *
      * @param ok true if the computation succeeded, false otherwise
      * @param metrics map of metrics about the computation
      * @param logs computation logs
-     * @param values result values of the senstivity computation. Must be non null.
+     * @param sensitivityValues result values of the senstivity computation in N
+     * @param sensitivityValuesContingencies result values of the senstivity computation on contingencies
      */
     @JsonCreator
     public SensitivityComputationResults(@JsonProperty("ok") boolean ok,
                                          @JsonProperty("metrics") Map<String, String> metrics,
                                          @JsonProperty("logs") String logs,
-                                         @JsonProperty("values") List<SensitivityValue> values) {
+                                         @JsonProperty("values") List<SensitivityValue> sensitivityValues,
+                                         @JsonProperty("contingencies present") boolean contingenciesPresent,
+                                         @JsonProperty("contingencyId values") Map<String, List<SensitivityValue>> sensitivityValuesContingencies) {
         this.ok = ok;
         this.metrics = Objects.requireNonNull(metrics);
         this.logs = Objects.requireNonNull(logs);
-        this.sensitivityValues = Objects.requireNonNull(values);
+        this.sensitivityValues = Objects.requireNonNull(sensitivityValues);
```

Comment:
Don't set directly the list but set as copy to prevent unsafe changes:
```java
this.sensitivityValues = Collections.unmodifiableList(Objects.requireNonNull(sensitivityValues));
```